code-challenges.test.js

```diff
@@ -14,33 +14,190 @@
+// To allow us to write a shorter function, use the arrow function syntax (=>).
+// Call on the argument of 'object' appended by a specific object value named ".name" which will run the name of each object key value.
+// Then append the value with the '.split()' built-in method followed by a set of parentheses with an enclosed set of quotations making sure to leave a space in between (a seperator to split between words). This will split the string of words into an array of substrings and return a new array.
+// Then we append the '.map' built-in method to this test to run the function for each key value in the array and pass in an argument named "value" in a set of parentheses, enclosed in another set of parentheses (to keep adding more code), which this test will provide the name value of the object key. 
+// To allow us to continue to write a shorter function, use the arrow function syntax (=>).
+// Call on the name value argument and assign the value, using the assignment operator (=) to run the name value and append it with the '.charAt' built-in method that will return a character at a specified index/position. We will add the index of '0' in a set of parentheses as an argument because the index of the first character is zero. 
+// To change the letter casing of the character index, we will apend the '.toUpperCase' built-in method because we want to change the first character to a capital letter followed by a set of empty parentheses (parameters are not required with this method).
+// Now we want to connect the value to the object key 'occupation by using the addition operator (+) and call of the 'value' argument and append the '.slice()' built-in method, which will start at a selected element in the array so we will pass in the index of '1' as an argument, enclosed by two sets of parentheses (to close out all of the previous logic up to this point) to run this test starting at the second character index. 
+// Next, we will append the '.join()' built-in method to join everything back together in the array and pass in quotation marks (seperator) in a set of parentheses, making sure to leave a space in between to join and space in between words.
+// To add a sentence to the code, us the addition operator (+) to connect the logic to the sentence. 
+// The sentence is called with a string interpolation method to insert different words into the string. We use a set of back-tics syntax (``) define the string, followed by the word 'is' (all the previous logic is only key value names so all we do here is add the key value occupation), a dollar symbol ($) followed by curly braces {}, within the curly braces, call on the initial object key, append the .occupation key value, followed by a period outside of the curly braces to end the sentence, close with the final back-tic and close with the final parentheses. 
+// Returns an array with a sentence about each person with their name capitalized.
```

Comment:
Evelyn, you went hard on this pseudo code! I love the enthusiasm, and all the right things are definitely in there. However, there are also a lot of things that don't need to be in there as well, such as syntax things, defining arguments and parameters, explaining what string interpolation is, etc. If you can remove the unnecessary material and keep in the parts that are specifically talking about the functionality of your code in clear, concise statements, you will be good to go!

Here's an example:
- input: array of objects
- output: string about the key value pairs with the name capitalized
- use .map on array to iterate on each object
- access the name with dot notation. use .split(" ") to turn the name into an array with each word as a string
- use .map on this new array to access each word. capitalize first letter of each word using .charAt(0) and .toUpperCase(), then concat the rest of the word in lower case using .slice(1)
- turn back to string using .join(" "), and concatenate with string including the object.occupation